javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,33 @@
+
+//create a button,input elements and image display div
+const userInput = document.getElementById('user-input');
+const limitOfImages = document.getElementById('limit');
+const button = document.getElementById('btn');
+const displayDiv = document.getElementById('display');
+
+//event listener for button click
+button.addEventListener('click', (event) => {
+    displayDiv.innerHTML = ''
+    const searchText = userInput.value;
+    const numberOfImagesToDisplay = limitOfImages.value;
+    //created an api key for fetching url 
+    fetch(`http://api.giphy.com/v1/gifs/search?q=${searchText}&api_key=pAvLnVXnC6fuqEVLzVF6FN3B8B78FLQN&limit=${numberOfImagesToDisplay}`)
+        .then(response => response.json())
+        .then(giphy => {
+            console.log(giphy);
```

Comment:
Better to remove all console.log after finish testing the program. Also from line 28

Reply:
Thanks..improved the code as per your suggestion